docker-compose-prod.yml

```diff
@@ -34,8 +35,9 @@ services:
      - default
 
 volumes:
+  static_files:
+    name: openoversight-static
   postgres:
-    driver: local
```

Comment:
Just out of curiosity, why remove this?

Reply:
It's redundant: https://docs.docker.com/compose/compose-file/compose-file-v3/#driver